docs/data/indexers/build-your-own/token-transfer-processor/README.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Token Transfer Processor
+sidebar_position: 0
+---
+
+## Overview
+
+The token transfer processor is a [package](https://github.com/stellar/go/tree/ttp-v1.0.0/ingest/processors/token_transfer) in the [ingest-sdk](../ingest-sdk/README.mdx) that tracks transfers, mints, burns, clawbacks, and fees for assets/tokens on the Stellar network.
```

Comment:
```suggestion
The token transfer processor is a [package](https://github.com/stellar/go/tree/ttp-v1.0.0/ingest/processors/token_transfer) which uses the [ingest-sdk](../ingest-sdk/README.mdx) to parse Stellar network transaction data and derive events representing token transfers, mints, burns, clawbacks, and fees.
```

it may be worthwhile to reference TTP as using ingest as a client rather than being part of it?